server/server.go

```diff
@@ -1033,8 +1033,7 @@ func (s *Server) v0QueryPath(w http.ResponseWriter, r *http.Request, urlPath str
 			}
 		}
 
-		partial, strictBuiltinErrors, instrument := false, false, false
-		rego, err := s.makeRego(ctx, partial, strictBuiltinErrors, txn, input, urlPath, m, instrument, nil, opts)
+		rego, err := s.makeRego(ctx, false, false, txn, input, urlPath, m, false, nil, opts)
```

Comment:
Skimming the file, it seems like the second parameter to `s.makeRego` is now always `false`. Would you mind removing it instead?

Reply:
The only reason I kept `s.makeRego` as-is is because of [this](https://github.com/open-policy-agent/opa/blob/main/server/server.go#L2516) code path. I guess we can remove that as well but thought it could be internally used in the future 🤷 WDYT?

Reply:
Hasn't been touched in a long time and it's still part of the git history. I'd drop it, it'll be less confusing than dead code.

Reply:
Updated